src/Pos/Types/Arbitrary.hs

```diff
@@ -42,14 +42,8 @@ import           Pos.Constants              (epochSlots, sharedSeedLength)
 import           Pos.Core.Address           (makePubKeyAddress, makeRedeemAddress,
                                              makeScriptAddress)
 import           Pos.Core.Coin              (coinToInteger, divCoin, unsafeSubCoin)
-import           Pos.Core.Types             (Address (..), ChainDifficulty (..), Coin,
-                                             CoinPortion, EpochIndex (..),
-                                             EpochOrSlot (..), LocalSlotIndex (..),
-                                             SharedSeed (..), SlotId (..), Timestamp (..),
-                                             getCoinPortion, mkCoin,
-                                             unsafeCoinPortionFromDouble, unsafeGetCoin)
-import           Pos.Core.Types             (ApplicationName (..), BlockVersion (..),
-                                             SoftwareVersion (..))
+import qualified Pos.Core.Types             as Types
```

Comment:
I am not sure that making, for example, `Coin` import qualified was a good idea.